src/arrays.js

```diff
@@ -1,61 +1,74 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const newIndex = index < array.length ? index : index - array.length;
+  return array[newIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split();
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  const newArray = [...array, element];
+  return newArray;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
+  return array;
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(item => item.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(item =>
+    item
+      .split('')
+      .reverse()
+      .join('')
+  );
 };
 
 const onlyEven = numbers => {
-  // your code here
+  return numbers.filter(number => number % 2 === 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
-};
-
-const elementsStartingWithAVowel = strings => {
-  // your code here
+  return array.filter((_, itemIndex) => itemIndex !== index);
```

Comment:
You also deleted the `elementsStartingWithAVowel` function here which is causing the test suite to fail as it cant find the function anymore!

To unblock you you could do this is several ways, one of which is to have an array of values to check for, then use a combination of the filter and includes array methods, and the toLowerCase string method:

```
const elementsStartingWithAVowel = strings => {
  const vowels = ['a', 'e', 'i', 'o', 'u'];

  return strings.filter(string => vowels.includes(string[0].toLowerCase()));
}
```